### ISSUE TITLE
Incorrect highlighting when broadcasting is used

### ISSUE THREAD

**Issue by ForceBru** [fonsp/Pluto.jl#1839]
Code:

```julia
for k ∈ eachindex(α)
	y = α[k] * normal_pdf.(x, μ[k], σ[k]^2)
	full_pdf .+= y
end
```

Pluto view:

![highlighting](https://user-images.githubusercontent.com/8695383/150574288-88c4a852-123a-48f7-8dee-2c838511e6d1.png)

1. Even though `normal_pdf` is defined, it's not underlined;
2. The name `full_pdf` is highlighted for no reason;
3. Placing the cursor after the closing parenthesis on the second line highlights only this single closing parenthesis, and not the opening parenthesis.

Same code after _removing the broadcasting operator_:

![highlighting_correct](https://user-images.githubusercontent.com/8695383/150574475-24705ebc-91bb-4911-b100-9ad34bebb9df.png)

1. `normal_pdf` is underlined
2. `full_pdf` is not highlighted
3. _Both_ opening and closing parentheses in `normal_pdf(x, μ[k], σ[k]^2)` are highlighted

-----

- Pluto v0.17.6 and the current master branch: https://github.com/fonsp/Pluto.jl/tree/90d0804da3850b6111304eb9d8af5a67758ac336
- Chrome 96.0.4664.110

**Comment by fonsp** [comment]
The incorrect highlighting of `full_pdf` is fixed after fixing #1836, but the underline problem is still there

**Comment by dralletje** [comment]
The lezer parser seems to struggle a lot with `func.()` even... on it

**Comment by dralletje** [comment]
@ForceBru by the way, your detailed issues are a blessing, thanks

**Comment by fonsp** [comment]
Maybe linked to #1865

**Comment by dralletje** [comment]
Fixed in Pluto 0.18!
